packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -50,6 +50,10 @@ export default defineComponent({
 	name: 'code-node-editor',
 	mixins: [linterExtension, completerExtension, workflowHelpers],
 	props: {
+		atCodeNode: {
```

Comment:
you could change this to `showAIButton` and update `ParameterInput` to pass in `:showAIButton = deploymentType === 'cloud'`, decoupling `CodeNodeEditor` from the settings store.